core/templates/dev/head/pages/exploration_player/CorrectnessFooterDirective.js

```diff
@@ -0,0 +1,36 @@
+// Copyright 2017 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for the correctness footer in the exploration player.
+ */
+
+oppia.directive('correctnessFooter', [
+  'UrlInterpolationService', function(UrlInterpolationService) {
+    return {
+      restrict: 'E',
+      scope: {},
+      templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
+        '/pages/exploration_player/' +
+        'correctness_footer_directive.html'),
+      controller: [
+        '$scope',
+        function($scope) {
+          $scope.correctnessfooterUrl = 
```

Comment:
If you can, find a way to do it using https://material.io/icons/#ic_check_circle -- that would save an image fetch. We use material icons elsewhere in the codebase, there should be plenty of examples.

This can be deferred to a separate PR if it's tricky, but in that case please file an issue for it.

Reply:
Since the color needs to be changed, I will create another issue once the PR gets in.